libs/product/src/models/bundled-product-item.ts

```diff
@@ -6,18 +6,23 @@ export enum DaffPriceTypeEnum {
 	dynamic = 'DYNAMIC'
 }
 
+export enum DaffBundledProductItemType {
```

Comment:
This is an enum not a type, the name should reflect this. It's also a `Composite` product.

Reply:
done